examples/stack/bqc/example_bqc_nv.py

```diff
@@ -0,0 +1,253 @@
+from __future__ import annotations
+
+import math
+import os
+from typing import Any, Dict, Generator
+
+import netsquid as ns
+from netqasm.lang.ir import BreakpointAction
+from netqasm.sdk.connection import BaseNetQASMConnection
+from netqasm.sdk.futures import Future, RegFuture
+from netqasm.sdk.qubit import Qubit
+
+from pydynaa import EventExpression
+from squidasm.run.stack.config import NVQDeviceConfig, StackNetworkConfig
+from squidasm.run.stack.run import run
+from squidasm.sim.stack.common import LogManager
+from squidasm.sim.stack.csocket import ClassicalSocket
+from squidasm.sim.stack.program import Program, ProgramContext, ProgramMeta
+
+
+class ClientProgram(Program):
+    PEER = "server"
+
+    def __init__(
+        self,
+        alpha: float,
+        beta: float,
+        trap: bool,
+        dummy: int,
+        theta1: float,
+        theta2: float,
+        r1: int,
+        r2: int,
+    ):
```

Comment:
Here I am not sure about the following variables what they mean:

* trap
* dummy
* r1
* r2

I assume that alpha, beta and both theta are just some other constants.